tests/conftest.py

```diff
@@ -315,6 +315,11 @@ def invoke_command(
     else:
         out_args = ["--out", out_file_name]
 
+    # The reason why we pass '--api-server' so then we can remove it and set it
+    # as part of the context is because we cannot pass it as an 'admin' option
+    # as we are supposed to do:
+    # https://github.com/pallets/click/issues/831#issuecomment-671048863,
+    # https://github.com/pallets/click/issues/832#issuecomment-671052776.
```

Comment:
I find it very confusing that you are inventing a new cli option (`--api-server` does exist on the subcommands)  just for calling this helper, especially when the option does exist on the base command).

Reply:
I have an idea how to improve it. Will propose another commit.

Reply:
I recommend using the `test_context` argument of your helper.